backend/lib/endpoints/posts.js

```diff
@@ -464,6 +470,7 @@ async function routes(app) {
       } else {
         body.expireAt = null;
       }
+      body.isEdited = true; // set edited true when update
```

Comment:
:+1:  you only need this, so you can remove it from the fluent-schema, because we don't want the user to input that.